mizar/common/common.py

```diff
@@ -460,3 +460,14 @@ def get_itf():
         return os.getenv("MIZAR_ITF")
     else:
         return default_itf
+
+def get_portal_host(core_api):
```

Comment:
Could we please have a description of what portal_host is and why we need it? Could it be named better?

Reply:
It is a gateway of a cluster. Since there has already been a gateway for endpoints, we name it as portal_host to transfer all the request traffics to external clusters.